ProjectApplication/project_core/filters.py

```diff
@@ -6,22 +6,24 @@
 
 
 class ProjectFilterSet(FilterSet):
-    key_title = CharFilter(method='filter_key_title', label="Key or Title")
-    keywords = CharFilter(method='filter_keywords')
+    key_title = CharFilter(method='filter_any', label="Key, Title, Keyword, PI, laysummary or organisation")
     geographical_areas = CharFilter(method='filter_geographical_areas')
     funding_instrument = CharFilter(method='filter_funding_instrument')
     start_date = CharFilter(method='filter_start_date')
 
-    def filter_key_title(self, queryset, name, value):
+    def filter_any(self, queryset, name, value):
         search_query = Q(
             Q(key__icontains=value) |
-            Q(title__icontains=value)
+            Q(title__icontains=value) |
+            Q(keywords__name__icontains=value) |
+            Q(principal_investigator__person__first_name__icontains=value) |
+            Q(principal_investigator__person__surname__icontains=value) |
+            Q(laysummary__text__icontains=value) |
+            Q(principal_investigator__organisation_names__name__icontains=value)
```

Comment:
Comment about the search fields:

I don't know if DarwinDigital discussed with Danièle/Basil which fields should be used by search (I haven't followed in detail all the conversations).

From a user point of view I would have expected that fields that are displayed (in https://spi.darwinstaging.com/projects/project-detail/?id=8b313787-e33e-4f34-878d-53879d5d2b2a ) to be possible to search. Quick ones that I'm missing:

- organisation
- regional focus
- location
- category? (maybe)

It would be nice to highlight the matched terms in the opened page as well?